src/main/java/core/basesyntax/FileWork.java

```diff
@@ -12,12 +12,11 @@
         try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
             StringBuilder wordsReader = new StringBuilder();
             String readLine = reader.readLine();
-            int counter = 0;
             while (readLine != null) {
                 String[] arrReadLine = readLine.split("\\s*(\\s|,|!|\\.|\\?)\\s*");
-                for (int i = 0; i < arrReadLine.length; i++) {
-                    if (arrReadLine[i].toLowerCase().startsWith(W_CHARACTER)) {
-                        wordsReader.append(arrReadLine[i].toLowerCase()).append(" ");
+                for (String arrReadLine1 : arrReadLine) {
```

Comment:
is `arrReadLine1` an informative name?

look through your code please, and check if all variable names are clear to understand if you read them in 2 weeks